db/comdb2.h

```diff
@@ -3614,4 +3613,6 @@ extern int gbl_disable_tpsc_tblvers;
 extern int gbl_osql_odh_blob;
 extern int gbl_pbkdf2_iterations;
 extern int gbl_bpfunc_auth_gen;
+extern int gbl_thread_count;
```

Comment:
We already have code to wait for threads to exit on clean-exit. https://github.com/bloomberg/comdb2/blob/master/db/thrman.c#L535

Why don't we just register these threads to the thread manager?

Reply:
Only some of the threads of our system go through thrman.c. 
Most of the ones that linger at the end are simply pthread_created so unfortunately I have to account for all of them myself. The upside is that if we structure our threads which loop to check for exit with a period of 1 second, then counting all of them is not necessary (ie counting a subset suffices because they exit at the same time because of the 1 per second check).

Reply:
I know. That's why I'm suggesting that you register these threads to the thread manager so it will take care of the wait-for-completion logic for you.

Reply:
I didn't understand you earlier, I can look into it I suppose.